frontend/simple/components/MembersCircle.vue

```diff
@@ -1,90 +1,197 @@
 <template>
-  <div>
-    <ul class='circle-container'>
-      <li v-for="(member, username) in members">
-        <img :src='member.attrs.picture' :style="member.style">
+  <div class="gi-memberCircle-container">
+    <svg viewBox="0 0 320 320" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="gi-svg">
+      <circle cx="160" cy="160" r="158" class="gi-svg-circle"/>
+    </svg>
+
+    <ul class="gi-circle-list">
```

Comment:
This view needs some `data-test` markers, so that, for example, the tests can verify whether or not the expected members are shown in the circle.

Reply:
Good point. What should the test cover here?

Reply:
Well, it should handle inviting over 6 people to the group. :)

But that can't be done until after that code actually works